wifiphisher/common/phishingpage.py

```diff
@@ -285,20 +285,23 @@ def __str__(self):
 class TemplateManager(object):
     """ This class handles all the template management operations """
 
-    def __init__(self):
+    def __init__(self, data_pages=None):
         """
         Construct object.
 
         :param self: A TemplateManager object
+        :param data_pages: The directory containing the templates
         :type self: TemplateManager
         :return: None
         :rtype: None
         """
 
         # setup the templates
-        self._template_directory = constants.PHISHING_PAGES_DIR
+        self._template_directory = data_pages or constants.PHISHING_PAGES_DIR
+        if data_pages:
+            constants.PHISHING_PAGES_DIR = data_pages
```

Comment:
`PHISHING_PAGES_DIR` is a constant and its value should never change. 

If you need it as a variable, you should switch it to lowercase.